friendshipper/src-tauri/src/repo/operations/gh/auto_merge_submit.rs

```diff
@@ -53,6 +53,26 @@ where
             return Err(CoreError::Input(anyhow!("No files to submit")));
         }
 
+        let status_op = StatusOp {
+            repo_status: self.repo_status.clone(),
+            app_config: self.app_config.clone(),
+            repo_config: self.repo_config.clone(),
+            engine: self.engine.clone(),
+            git_client: self.git_client.clone(),
+            github_username: self.github_client.username.clone(),
+            aws_client: None,
+            storage: None,
+            allow_offline_communication: false,
+            skip_display_names: true,
+
+            // we'll make sure this gets done at the end
+            skip_engine_update: true,
```

Comment:
save ~7 seconds against unreal editor - we'll look into why it takes that long separately.